install/infra/modules/aks/local.tf

```diff
@@ -14,9 +14,9 @@ locals {
   ])
 
   workspace_name = replace(terraform.workspace, "/[\\W\\-]/", "") # alphanumeric workspace name
-  db       = "GP_Gen5_2"
-  location = substr(var.location, 0, 3) # Short code for location
-  machine  = "Standard_D4_v3"
+  db             = "GP_Gen5_2"
```

Comment:
**thought(style):** while slipping in style fixes in smaller pull requests is no big deal, this is a pretty hefty pull request and style fixups can increase the cost of reviewing. I'll absolutely rubber stamp pull request that just runs code reformatting so if you're in a situation where code needs to be reformatted consider extracting a quick format PR so we can iterate faster on the fun stuff. 😊

Reply:
Sorry for the confusion here. I have kind of accumulated all the lint changes on [this PR ](https://github.com/gitpod-io/gitpod/pull/13541), I should have put this PR on draft to avoid the risk of it getting reviewed.